src/components/lec6/lec6task6/apierror.js

```diff
@@ -0,0 +1,50 @@
+
+import React, { Component } from "react";
+
+
+export default class ApiError extends Component {
+    constructor(props){
+        super(props);
+
+        this.state = {
+            posts: [],
+            isLoading: false,
+            error: null,
+          };
+    }
+
+    componentDidMount() {
+        this.setState({ isLoading: true });
+
+        fetch('https://github.com/kkmkkkkk')
+          .then(response => response.json())
+          .then(data => this.setState({ posts: data, isLoading: false }))
+          .catch(error => this.setState({ error, isLoading: false }));
```

Comment:
You shouldn't store the whole error object in the state instead the `error.message` value except you need it which in most case you don't.